### DISCUSSION TITLE
blog/istio-pod-process-lifecycle/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#299] [Announcements]
# blog/istio-pod-process-lifecycle/

本文将为你讲解 Istio 的 Init 容器、Pod 内部进程及启动过程。

https://jimmysong.io/blog/istio-pod-process-lifecycle/

**Comment by FesonX** [comment]
> 关于 Istio 中端口用途请参考 [Istio 官方文档](https://istio.io/latest/zh/docs/ops/deployment/requirements/)。

此处的链接已变更为 https://istio.io/latest/docs/ops/deployment/application-requirements/

**Reply by rootsongjc** [reply]
感谢反馈，链接已修改。
